src/main/scala/tech/cryptonomic/conseil/util/JsonUtil.scala

```diff
@@ -56,7 +56,7 @@ object JsonUtil {
     lazy val emptyObject = JsonString("{}")
 
     /** add standard cleaning for input json */
-    def sanitize(s: String): String = s.filterNot(_.isControl).replaceAll("""\\u[[a-zA-Z0-9]]{1,4}""", "[REDACTED_BY_CONSEIL]")
+    def sanitize(s: String): String = s.filterNot(_.isControl).replaceAll("""\\u[[a-zA-Z0-9]]{1,4}""", "REDACTED_BY_CONSEIL")
```

Comment:
Why the double brackets? Did I suggest that?

Shouldn't the regex be `"""\\u[a-zA-Z0-9]{1,4}"""` ?

Reply:
And now that I come to look better at it

`"""\\u[a-fA-F0-9]{1,4}"""` should be more lenient, we where redacting things that couldn't have been unicode values